### PR TITLE
Added swhub

### PR DESCRIPTION
**Pull request by MatthiasLettner**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by MatthiasLettner** [comment]
my servername is the directory name

File: minecraft_servers/swhub/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "server_name": "SWHub",
```
Comment (maaaathis) [inline minecraft_servers/swhub/manifest.json:2]: `server_name` must be lowercase

File: minecraft_servers/swhub/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "nice_name": "SWHub",
+  "direct_ip": "swhub.net",
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://swhub.net",
+    "web_shop": "https://lodianet.tebex.io",
```
Comment (maaaathis) [inline minecraft_servers/swhub/manifest.json:10]: Returns error 404

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/swhub/manifest.json (hunk 1 above)
Comment (DoJapHD) [inline minecraft_servers/swhub/manifest.json:2]: Change to all lowercase

File: minecraft_servers/swhub/manifest.json (hunk 2 above)
Comment (DoJapHD) [inline minecraft_servers/swhub/manifest.json:10]: 404

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by DoJapHD** [comment]
Oh well, you already did that matthis xd

**Comment by maaaathis** [comment]
> Oh well, you already did that matthis xd
sry haha

**Pull request closed without merging**